### ISSUE TITLE
Use generic temporaryName property instead of specific ones

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-gnd#227]
As with `preferredName`, we should only use `temporaryName` in lobid-gnd data for all these properties: `temporaryName`, `temporaryNameOfTheConferenceOrEvent`, `temporaryNameOfTheCorporateBody`, `temporaryNameOfThePlaceOrGeographicName`.

**Comment by fsteeg** [comment]
That's a good idea too, but what we discussed was about `abbreviatedNameForThe*`.

**Comment by acka47** [comment]
> That's a good idea too, but what we discussed was about `abbreviatedNameForThe*`.

Yes, but we can already implement this as a generic property exists. For `abbreviatedNameForThe* I will first open an issue at DNB to add the generic property in the first place.

**Comment by fsteeg** [comment]
After looking at this, I now think https://github.com/hbz/lobid-gnd/issues/229, like this here, is actually about using the generic property (abbreviatedName there, temporaryName here), and filling it with values that in the DNB data are stored in the specific fields. Given that we have some unification with the DNB data on the horizon, and this has not been driven by a concrete user request, I think we should not invest the time to implement this right now.

**Comment by acka47** [comment]
Ok. Then we can close this issue for now?

**Comment by fsteeg** [comment]
Yes, good catch! We only had `temporaryName` (which [isn't used](https://lobid.org/gnd/search?q=temporaryName%3A*) in the data). Added the specific ones in https://github.com/hbz/lobid-gnd/commit/bd52135562e0c5a6a02fba259986fb7957e0eb60.

**Comment by acka47** [comment]
As discussed offline, we aim at the consistent approach for all name properties, i.e. using `*Name` only and getting rid of specific versions like `*NameOfTheCorporateBody` but will not implement this right now.
